src/control/LayerManager.ts

```diff
@@ -145,6 +145,21 @@ export class LayerManager {
     return this;
   }
 
+  setZoomLevel(zoomlevels: [number, number]): LayerManager {
```

Comment:
Maybe call this something else like setZoomLevelLimits or setMinMaxZoomLevel, my first thought when I see setZoomLevel is a function I can use to set the current zoom level

Reply:
renamed it to setZoomLevelBoundary, hope that will cover it